pager/src/main/java/com/google/accompanist/pager/PagerIndicator.kt

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright 2021 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.accompanist.pager
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.offset
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.Shape
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+
+/**
+ * Contains default values used for [PagerIndicator].
+ */
+object PagerDefaults {
+    /**
+     * Default spacing used for [PagerIndicator].
+     */
+    val Spacing = 8.dp
+
+    /**
+     * Default indicator size used for [PagerIndicator].
+     */
+    val Size = 8.dp
+
+    /**
+     * Default shape used for [PagerIndicator].
+     */
+    val Shape = CircleShape
+
+    /**
+     * Default Indicator color value used for [PagerIndicator].
+     */
+    val IndicatorColor = Color.White
+
+    /**
+     * Default alpha value used for generating the inactive indicators color for [PagerIndicator].
+     */
+    val InactiveIndicatorColorAlpha = 0.6f
+}
+
+/**
+ * A Indicator for a [Pager] representing the currently active page and total pages using a [Shape].
+ *
+ * This Element allows the setting of both the [shape] and [indicatorShape], which defines how the
+ * Indicator is visually represented. By default Indicators are represented as [CircleShape].
+ * When changing the [shape] by default the [indicatorShape] adjusts accordingly. If you want the
+ * active page indicator to have a different appearance override [indicatorShape].
+ *
+ * @sample com.google.accompanist.sample.pager.PagerIndicatorSample
+ *
+ * @param pagerState the state object of your [Pager] to be used to observe the list's state.
+ * @param modifier the modifier to apply to this layout.
+ * @param indicatorColor the color of the active Page indicator
```

Comment:
Would names like activeColor, disabledColor be better?

Reply:
I had similar thoughts already although "disabled" might not be correct. Also not sure if "inactive" is a better replacement